src/core/pointcloud/qgspointcloudclassifiedrenderer.cpp

```diff
@@ -149,16 +149,13 @@ void QgsPointCloudClassifiedRenderer::renderBlock( const QgsPointCloudBlock *blo
       }
 
       if ( renderAsTriangles() )
-      {
         addPointToTriangulation( x, y, z, color, context );
-      }
-      else
-      {
-        const double size = pointSizes.value( attributeValue );
-        drawPoint( x, y, color, size, context );
-        if ( renderElevation )
-          drawPointToElevationMap( x, y, z, size, context );
-      }
+
+      const double size = pointSizes.value( attributeValue );
+      drawPoint( x, y, color, size, context );
```

Comment:
How's this work with semi transparent triangles? There's a mechanism in the tiled scene renderer for this kind of temporary results which should be used instead

Reply:
You're right, large points are also visible around the triangles' outer edges as in the failing tests.
I'm reverting this change and properly handle that on a separate PR as you suggest.

Reply:
Added this on #56134